hms-api/src/service/idea-comment-service.ts

```diff
@@ -32,12 +30,6 @@ export async function createIdeaComment(
     throw new ReferenceNotFoundError(`User with id: ${userId} not found`);
   }
 
-  if (parentCommentId && !(await parentCommentIdIdeaExists(parentCommentId))) {
-    throw new ReferenceNotFoundError(
-      `Parent comment with id: ${parentCommentId} not found`,
-    );
-  }
```

Comment:
why is this removed?

Reply:
for consistency as we nowwhere else check for partencommentId neither test for it 
just to have it completly removed for now until we someday add partenCommentIds

Reply:
then I would argue that you should either leave it (so that you don't have to remember later to put it back) or you remove everything related to parent comments

Reply:
just think about as it was never there

Reply:
but, it IS there.  In the function arguments.  There is no reason to not just leave this here